tox.ini

```diff
@@ -13,19 +13,18 @@ setenv =
     DJANGO_SETTINGS_SKIP_LOCAL=True
 passenv = CI TRAVIS TRAVIS_* HOME
 deps =
-  -r{toxinidir}/requirements/testing.txt
-  debug: -r{toxinidir}/requirements/debug.txt
-changedir = {toxinidir}/readthedocs
+  -r requirements/testing.txt
+  debug: -r requirements/debug.txt
 basepython =
     python3.8
 commands =
     /bin/sh -c '\
         export DJANGO_SETTINGS_MODULE=readthedocs.settings.test; \
-        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --suppress-no-test-exit-code -m "not proxito and not embed_api" {posargs:{env:TOX_POSARGS:-m "not search and not proxito and not embed_api"}}'
+        pytest --cov-report= --cov-config .coveragerc --cov=. --pyargs readthedocs --suppress-no-test-exit-code -m "not proxito and not embed_api" {posargs:{env:TOX_POSARGS:-m "not search and not proxito and not embed_api"}}'
 
     /bin/sh -c '\
         export DJANGO_SETTINGS_MODULE=readthedocs.settings.proxito.test; \
-        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --cov-append -m proxito --suppress-no-test-exit-code {posargs}'
+        pytest --cov-report= --cov-config .coveragerc --cov=. --cov-append --pyargs readthedocs -m proxito --suppress-no-test-exit-code {posargs}'
```

Comment:
this is the way to run our tests (using --pyargs) https://docs.pytest.org/en/7.0.x/explanation/goodpractices.html#tests-as-part-of-application-code